Archives/ClmFile.cpp

```diff
@@ -80,11 +81,11 @@ namespace Archives
 
 			waveFileWriter.Write(header);
 
-			std::unique_ptr<SeekableStreamReader> reader = clmFileReader.Slice(
+			FileSliceReader reader = clmFileReader.Slice(
```

Comment:
We should use `auto` here to protect against return type changes in the FileStreamReader code, particularly since the API is not yet stable.